LICOR.py

```diff
@@ -342,10 +401,12 @@ def flux_calculation(fluxes, CO2_or_CH4):
 
         flux.RSQ = R2
         flux.RoC = m * 60
-        if CO2_or_CH4.lower() == "co2":
+        if LICOR_GAS == "co2":
             flux.flux = (flux.RoC*(vol/(0.0821*flux.temp))*(0.044*1440)/(flux.surface_area)*(12/44)/1000)
-        else:
```

Comment:
Change this formula slightly so that the output units for CH4 are mg instead of g. This is reflected in the `output_units` variable defined on line 34.